crossing-training.lic

```diff
@@ -382,27 +382,51 @@ class CrossingTraining
   def calculate_mana(min, more, discern_data, cyclic)
     total = min + more
     total = (total * @settings.prep_scaling_factor).floor
-    discern_data['mana'] = [(total / 4.0).ceil, min].max
+    discern_data['mana'] = [(total / 5.0).ceil, min].max
     remaining = total - discern_data['mana']
-    if remaining > @settings.cambrinth_cap
-      discern_data['mana'] = discern_data['mana'] + (remaining - @settings.cambrinth_cap)
+    if @settings.cambrinth_items
+      cambrinth_items = @settings.cambrinth_items
+    else
+      cambrinth_items = []
+      cambrinth_items[0] = {}
+      cambrinth_items[0]['name'], cambrinth_items[0]['cap'], cambrinth_items[0]['stored'] = map_cambrinth(@settings)
```

Comment:
Did you consider using this syntax for making a new Hash?

```
cambrinth_items[0] = {
  'name' => @settings.cambrinth,
  'cap' => @settings.cambrinth_cap
  'stored' => @settings.stored_cambrinth
}
```

Reply:
Since I was assigning it in a separate section also, I figured it made sense to define a new method to map it, though I don't know how much more efficient it is, since it's only a few lines and I had to declare the array and hash before calling `map_cambrinth`, anyway.  I'm fine switching it back if you think it makes more sense.

Reply:
Personally, I prefer the style I suggested, but it's not a big deal.

Reply:
You can do it all on one line really, 
```
cambrinth_items = [{
  'name' => @settings.cambrinth,
  'cap' => @settings.cambrinth_cap
  'stored' => @settings.stored_cambrinth
}]
```